token-metadata/program/src/utils/master_edition.rs

```diff
@@ -46,6 +47,7 @@ pub fn process_mint_new_edition_from_master_edition_via_token_logic<'a>(
     accounts: MintNewEditionFromMasterEditionViaTokenLogicArgs<'a>,
     edition: u64,
     ignore_owner_signer: bool,
```

Comment:
Can we just get rid of this as it appears to be set to `false` all the time in the code?

Reply:
I believe this is an argument in the old edition printing instruction.

Reply:
Yes I agree but it is hardcoded to `false` there as well.   So it is never used in any of the code, it is always `false`.